### PR TITLE
Tips for settings and fixed urls

### PR DESCRIPTION
**Pull request by BychekRU**

### CODE REVIEW

File: pontoon/base/templates/widgets/checkbox.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{% macro checkbox(label, class, attribute, is_enabled=False) %}
-<li class="check-box {{ class }}{% if is_enabled %} enabled{% endif %}" data-attribute="{{ attribute }}"><i class="fa fa-fw"></i>{{ label }}</li>
+{% macro checkbox(label, class, attribute, is_enabled=False, title=False) %}
+<li class="check-box {{ class }}{% if is_enabled %} enabled{% endif %}" data-attribute="{{ attribute }}"{% if title %} title = "{{title}}"{% endif %}><i class="fa fa-fw"></i>{{ label }}</li>
```
Comment (mathjazz) [inline pontoon/base/templates/widgets/checkbox.html]: Remove space surrounding `=` in  `title = `.

File: pontoon/base/templates/translate.html [hunk 2]
```diff
@@ -286,9 +286,9 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
             <div class="selector fa fa-cog" title="Settings"></div>
             <div class="menu">
               <ul>
-                {{ Checkbox.checkbox('Quality Checks', class='quality-checks', attribute='quality_checks', is_enabled=user.profile.quality_checks) }}
+                {{ Checkbox.checkbox('Quality Checks', class='quality-checks', attribute='quality_checks', is_enabled=user.profile.quality_checks, title='Allows you turn off checks before submitting your translations') }}
```
Comment (mathjazz) [inline pontoon/base/templates/translate.html]: Let's change text to:
Run quality checks before submitting translations

File: pontoon/base/templates/translate.html [hunk 3]
```diff
@@ -286,9 +286,9 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
...
                 {% if user.can_translate(project=project, locale=locale) %}
-                {{ Checkbox.checkbox('Make Suggestions', class='force-suggestions', attribute='force_suggestions', is_enabled=user.profile.force_suggestions) }}
+                {{ Checkbox.checkbox('Make Suggestions', class='force-suggestions', attribute='force_suggestions', is_enabled=user.profile.force_suggestions, title='Lets save translations at once without making suggestions') }}
```
Comment (mathjazz) [inline pontoon/base/templates/translate.html]: Let's change text to:
Save suggestions instead of translations

**Comment by BychekRU** [comment]
@mathjazz Thanks for notes! Done.

**Pull request merged**